tests/js/spec/components/modals/widgetViewerModal.spec.tsx

```diff
@@ -110,6 +110,15 @@ describe('Modals -> WidgetViewerModal', function () {
     it('renders Discover area chart widget viewer', function () {
       expect(container).toSnapshot();
     });
+
+    it('redirects user to Discover when clicking Open in Discover', async function () {
+      const openInDiscoverButton = await screen.findByText('Open in Discover');
+      expect(openInDiscoverButton).toBeInTheDocument();
+      expect(openInDiscoverButton.closest('a')).toHaveAttribute(
```

Comment:
in this case we could also write

```suggestion
      expect(screen.getByRole('link', {name: 'Open in Discover'})).toHaveAttribute(
      'href',
      '/organizations/org-slug/discover/results/?field=count%28%29&field=failure_count%28%29&name=Test%20Widget&query=title%3A%2Forganizations%2F%3AorgId%2Fperformance%2Fsummary%2F&statsPeriod=14d&yAxis=count%28%29&yAxis=failure_count%28%29'
    );
```

Reply:
👍 updated this to use `findByLabelText` since the `Button` component has the url now and it also has a label.

Reply:
in this case, shouldn't we give priority to `getByRole('button', {...})`? Please see https://testing-library.com/docs/queries/about/#priority